pkg/drivers/pgsql/pgsql.go

```diff
@@ -41,14 +42,19 @@ var (
  				value bytea,
  				old_value bytea
  			);`,
+
 		`CREATE INDEX IF NOT EXISTS kine_name_index ON kine (name)`,
 		`CREATE INDEX IF NOT EXISTS kine_name_id_index ON kine (name,id)`,
 		`CREATE INDEX IF NOT EXISTS kine_id_deleted_index ON kine (id,deleted)`,
 		`CREATE INDEX IF NOT EXISTS kine_prev_revision_index ON kine (prev_revision)`,
 		`CREATE UNIQUE INDEX IF NOT EXISTS kine_name_prev_revision_uindex ON kine (name, prev_revision)`,
+		`CREATE INDEX IF NOT EXISTS kine_list_query_index on kine(name, id DESC, deleted)`,
 	}
 	schemaMigrations = []string{
 		`ALTER TABLE kine ALTER COLUMN id SET DATA TYPE BIGINT, ALTER COLUMN create_revision SET DATA TYPE BIGINT, ALTER COLUMN prev_revision SET DATA TYPE BIGINT; ALTER SEQUENCE kine_id_seq AS BIGINT`,
+		// It is important to set the collation to "C" to ensure that LIKE and COMPARISON
+		// queries use the index.
+		`ALTER TABLE kine ALTER COLUMN name SET DATA TYPE TEXT USING name::TEXT COLLATE "C"`,
```

Comment:
Would you mind adding an empty second schema migration to the mysql driver? For drivers that have schema migrations, we should ensure that they are either no-ops or have the same effect across all drivers for the sake of consistency. I wouldn't want the next migration to be 3rd for postgres but 2nd for mysql.

Reply:
Sure - let me do that.